### ISSUE TITLE
[api-extractor] Internal Error: Unable to determine semantic information for declaration

### ISSUE THREAD

**Issue by bpasero** [microsoft/rushstack#3738]
## Summary

I am trying to use `api-extractor` to produce a `web.api.d.ts` file for the VS Code web embedder API. I ran the tool and it resulted in this message that makes me file this issue:

`You have encountered a software defect. Please consider reporting the issue to the maintainers of this application.`

## Repro steps

* `git clone https://github.com/microsoft/vscode.git`
* `cd vscode`
* `api-extractor init`
* set `"projectFolder": "src"`
* set `"mainEntryPointFilePath": "./src/vs/workbench/browser/web.api.d.ts"`
* `tsc ./src/vs/workbench/browser/web.api.ts --declaration --emitDeclarationOnly`
* `api-extractor run --local --diagnostics`
* see an error in output

```
ERROR: Internal Error: Unable to determine semantic information for declaration:
/Users/bpasero/Development/Microsoft/vscode/src/vs/base/common/uri.ts:220:7
```

## Standard questions

Please answer these questions to help us investigate your issue more quickly:

| Question | Answer |
| -------- | -------- |
| `@microsoft/api-extractor` version? | 7.33.6 |
| Operating system? | Mac |
| API Extractor scenario? | rollups (.d.ts) |
| TypeScript compiler version? | 4.8.4 |
| Node.js version (`node -v`)? | 16.16.0 |

**Comment by zelliott** [comment]
Thanks for filing the issue @bpasero. When I run:

`tsc ./src/vs/workbench/browser/web.api.ts --declaration --emitDeclarationOnly`

I get a bunch of module resolution errors. For example:


> vs/workbench/browser/web.api.ts:6:38 - error TS2307: Cannot find module 'vs/base/common/performance' or its corresponding type declarations.
> import type { PerformanceMark } from 'vs/base/common/performance';

What am I doing wrong?

**Comment by bpasero** [comment]
Yeah I see that too, but there is still a `d.ts` file created so I had thought I would proceed with the process, ignoring these errors for now. Do you think its related?
